Nodejs/Setup/setup.proj

```diff
@@ -19,6 +19,7 @@
     <ProjectFile Include="NodejsTools\NodejsTools.wixproj"/>
     <ProjectFile Include="NodejsToolsInstaller\NodejsToolsInstaller.wixproj"/>
     <ProjectFile Include="..\..\Internal\Nodejs\VsLoggerSetup\VsLogger.wixproj" Condition="'$(IncludeVsLogger)' == 'True'"/>
+    <ProjectFile Include="InstallerTelemetryAction\InstallerTelemetryAction.csproj" Condition="'$(ReleaseBuild)' == 'True'" />
```

Comment:
Thanks, I added a variable ReleaseBuild in .wxs file to determine if its a release build or not. hence include the custom action or not.